packages/hawtio/src/plugins/connect/jolokia-service.ts

```diff
@@ -421,8 +422,7 @@ class JolokiaService implements IJolokiaService {
   }
 
   loadUpdateRate(): number {
-    const updateRate = localStorage.getItem(STORAGE_KEY_UPDATE_RATE)
-    return updateRate ? parseInt(updateRate) : DEFAULT_UPDATE_RATE
+    return preferencesService.getJolokiaUpdateRate();
```

Comment:
Do not delegate the method to preferencesService. Keep the implementation in this class.